go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/dgryski/go-bitstream v0.0.0-20180413035011-3522498ce2c8
-	github.com/docker/docker v1.13.1 // indirect
+	github.com/docker/docker v1.13.1
```

Comment:
any ideas why this changed?

Reply:
I have no idea. It was failing in CI, so I ran `go mod tidy` and it added that comment, and now CI passes.

Reply:
Oh, it's because my IDE chose the `ioutils` package from Docker. I wanted the Go stdlib one. I fixed it.